card/mod/04_settings/set/type/setting.rb

```diff
@@ -1,5 +1,13 @@
 require_dependency 'json'
-  
+
+def self.member_names
+  @@member_names ||= begin
+    Card.search( :type_id=>SettingID, :return=>'key' ).inject({}) do |hash, card_key|
```

Comment:
I used to use inject like this until a colleague kept pointing out that each_with_object is better in this case (assuming we have it in ruby 1.9.3).  It only works when your block in changing the object via side-effects, which you are in this case.